src/components/Table/Table.tsx

```diff
@@ -68,11 +68,17 @@ const TableRow = styled.tr`
   }
 
   @media (max-width: 768px) {
+    position: relative;
     display: flex;
     flex-wrap: wrap;
-    ${({ theme }) => `
+    ${({ theme, $selectable = false }) => `
       border: 1px solid ${theme.click.table.row.color.stroke.default};
       border-radius: ${theme.click.table.radii.all};
+      ${
+        $selectable
+          ? `padding-left: calc(${theme.click.table.body.cell.space.sm.x} + ${theme.click.table.body.cell.space.sm.x} + 1rem);`
```

Comment:
why do we need to have `1rem` hardcoded?

Reply:
Fixed it